packages/react/src/constants/LayoutConstants.js

```diff
@@ -9,6 +9,7 @@ export const COLORS = {
 };
 
 export const CARD_SIZES = {
+  XSMALL: 'XSMALL',
```

Comment:
is this supported ? As far as I'm aware support for XSMALL was removed. and when ever someone use XSMALL we used to change it to 'SMALL' 

This may have impact on monitor dashboards.

Reply:
@herleraja Hello.  Support for xsmall was removed, but we have a scenario in Maximo where the use of it is required again. It will only be allowed for certain simple Value cards, and there is likely some CSS cleanup and other work that will need to go in to that card type.  What would work for Monitor here? Is there a condition or check we could add to isolate you from this change? @anishkumar-bhut @hectordavis @stuckless

Reply:
@JordanWSmith15 Just checked. I think the monitor should not be affected by this change. 

We override the props accordingly. 
 
![image](https://github.com/user-attachments/assets/d98c2f2f-996e-4a4e-930d-6ea89ce37485)

Reply:
@hectordavis the warning thrown in getUpdatedCardSize but that's thrown only in development mode. It should not effect in production. 
![Uploading image.png…]()